### ISSUE TITLE
Getting Error when running the Workflow

### ISSUE THREAD

**Issue by Eric-EWeb** [Tavris1/ComfyUI-Easy-Install#279]
**NOTE**: The workflow runs fine when I turn off the "Dynamic VRAM", but only for the first few generations, after that it gives the "OOM" errors no matter how many times I "Unload" the models or "FREE" the model cache

I got the issue only today, everything was working perfectly file till I ran the "Update Easy-Install.bat" and the "Update ComfyUI.bat"

Any help will be highly appreciated.

### LOG

[INFO] Model Krea2TEModel_ prepared for dynamic VRAM loading. 4999MB Staged. 0 patches attached. Force pre-loaded 249 weights: 627 KB.
[INFO] Model Krea2TEModel_ prepared for dynamic VRAM loading. 4999MB Staged. 0 patches attached. Force pre-loaded 249 weights: 627 KB.
[INFO] Requested to load Krea2
[INFO] Model Krea2 prepared for dynamic VRAM loading. 12864MB Staged. 256 patches attached. Force pre-loaded 160 weights: 2824 KB.
  0%|                                                                | 0/8 [00:00<?, ?it/s,   Model Initializing ...  ]Fatal Python error: Aborted

Stack (most recent call first):
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\ldm\krea2\model.py", line 79 in forward
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1787 in _call_impl
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1776 in _wrapped_call_impl
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\ldm\krea2\model.py", line 214 in forward
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1787 in _call_impl
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1776 in _wrapped_call_impl
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\ldm\krea2\model.py", line 370 in _forward
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\ldm\krea2\model.py", line 277 in forward
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1787 in _call_impl
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\nn\modules\module.py", line 1776 in _wrapped_call_impl
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\model_base.py", line 240 in _apply_model
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\model_base.py", line 196 in apply_model
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 335 in _calc_cond_batch
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 219 in _calc_cond_batch_outer
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 211 in calc_cond_batch
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 620 in sampling_function
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1218 in predict_noise
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1215 in outer_predict_noise
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1208 in __call__
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 640 in __call__
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\k_diffusion\sampling.py", line 205 in sample_euler
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\python_embeded\Lib\site-packages\torch\utils\_contextlib.py", line 124 in decorate_context
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1005 in sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1235 in inner_sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1260 in outer_sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\patcher_extension.py", line 113 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1333 in sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1351 in sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\samplers.py", line 1461 in sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\comfy\sample.py", line 74 in sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\nodes.py", line 1580 in common_ksampler
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\nodes.py", line 1616 in sample
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\execution.py", line 306 in process_inputs
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\execution.py", line 318 in _async_map_node_over_list
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\execution.py", line 344 in get_output_data
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\execution.py", line 545 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\execution.py", line 789 in execute_async
  File "asyncio\events.py", line 88 in _run
  File "asyncio\base_events.py", line 1999 in _run_once
  File "asyncio\base_events.py", line 645 in run_forever
  File "asyncio\windows_events.py", line 322 in run_forever
  File "asyncio\base_events.py", line 678 in run_until_complete
  File "asyncio\runners.py", line 118 in run
  File "asyncio\runners.py", line 195 in run
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\execution.py", line 728 in execute
  File "E:\Easy ComfyUI\ComfyUI-Easy-Install\ComfyUI\main.py", line 372 in prompt_worker
  File "threading.py", line 1012 in run
  File "threading.py", line 1075 in _bootstrap_inner
  File "threading.py", line 1032 in _bootstrap

Extension modules: sqlalchemy.cyextension.collections, sqlalchemy.cyextension.immutabledict, sqlalchemy.cyextension.processors, sqlalchemy.cyextension.resultproxy, sqlalchemy.cyextension.util, greenlet._greenlet, markupsafe._speedups, yaml._yaml, PIL._imaging, multidict._multidict, yarl._quoting_c, propcache._helpers_c, aiohttp._http_writer, aiohttp._http_parser, aiohttp._websocket.mask, aiohttp._websocket.reader_c, frozenlist._frozenlist, charset_normalizer.md, charset_normalizer.cd, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator, torch._C, torch._C._dynamo.autograd_compiler, torch._C._dynamo.eval_frame, torch._C._dynamo.guards, torch._C._dynamo.utils, torch._C._fft, torch._C._linalg, torch._C._nested, torch._C._nn, torch._C._sparse, torch._C._special, psutil._psutil_windows, PIL._imagingft, _cyutility, scipy._cyutility, scipy._lib._ccallback_c, scipy.ndimage._nd_image, scipy.ndimage._rank_filter_1d, scipy.special._ufuncs_cxx, scipy.special._ellip_harm_2, scipy.special._special_ufuncs, scipy.special._gufuncs, scipy.special._ufuncs, scipy.special._specfun, scipy.special._comb, _ni_label, scipy.ndimage._ni_label, av._core, av.logging, av.bytesource, av.buffer, av.audio.format, av.error, av.dictionary, av.container.pyio, av.option, av.descriptor, av.format, av.utils, av.stream, av.container.streams, av.sidedata.motionvectors, av.sidedata.sidedata, av.opaque, av.packet, av.container.input, av.container.output, av.container.core, av.codec.context, av.video.format, av.video.reformatter, av.plane, av.video.plane, av.video.frame, av.video.stream, av.codec.hwaccel, av.codec.codec, av.frame, av.audio.layout, av.audio.plane, av.audio.frame, av.audio.stream, av.filter.link, av.filter.context, av.filter.graph, av.filter.filter, av.filter.loudnorm, av.audio.resampler, av.audio.codeccontext, av.audio.fifo, av.bitstream, av.video.codeccontext, regex._regex, scipy.integrate._odepack, scipy.integrate._quadpack, scipy.integrate._vode, scipy.integrate._dop, scipy.sparse._sparsetools, _csparsetools, scipy.sparse._csparsetools, scipy.linalg._fblas, scipy.linalg._flapack, scipy.linalg.cython_lapack, scipy.linalg._cythonized_array_utils, scipy.linalg._solve_toeplitz, scipy.linalg._batched_linalg, scipy.linalg._decomp_lu_cython, scipy.linalg._matfuncs_schur_sqrtm, scipy.linalg._matfuncs_expm, scipy.linalg._linalg_pythran, scipy.linalg.cython_blas, scipy.linalg._decomp_update, scipy.sparse.linalg._dsolve._superlu, scipy.sparse.linalg._eigen.arpack._arpacklib, scipy.sparse.linalg._propack, scipy.optimize._group_columns, scipy._lib.messagestream, scipy.optimize._trlib._trlib, scipy.optimize._lbfgsb, _moduleTNC, scipy.optimize._moduleTNC, scipy.optimize._slsqplib, scipy.optimize._minpack, scipy.optimize._lsq.givens_elimination, scipy.optimize._zeros, scipy._lib._uarray._uarray, scipy.linalg._decomp_interpolative, scipy.optimize._bglu_dense, scipy.optimize._lsap, scipy.spatial._ckdtree, scipy.spatial._qhull, scipy.spatial._voronoi, scipy.spatial._hausdorff, scipy.spatial._distance_wrap, scipy.spatial.transform._rotation_cy, scipy.spatial.transform._rigid_transform_cy, scipy.optimize._direct, scipy.interpolate._fitpack, scipy.interpolate._dfitpack, scipy.interpolate._dierckx, scipy.interpolate._ppoly, scipy.interpolate._interpnd, scipy.interpolate._rbfinterp_pythran, scipy.interpolate._rgi_cython, scipy.special.cython_special, scipy.stats._stats, scipy.stats._biasedurn, scipy.stats._stats_pythran, scipy.stats._levy_stable.levyst, scipy.stats._ansari_swilk_statistics, scipy.sparse.csgraph._tools, scipy.sparse.csgraph._shortest_path, scipy.sparse.csgraph._traversal, scipy.sparse.csgraph._min_spanning_tree, scipy.sparse.csgraph._flow, scipy.sparse.csgraph._matching, scipy.sparse.csgraph._reordering, scipy.stats._sobol, scipy.stats._qmc_cy, scipy.stats._rcont.rcont, scipy.stats._qmvnt_cy, av.subtitles.stream, scipy.signal._sigtools, scipy.signal._max_len_seq_inner, scipy.signal._upfirdn_apply, scipy.signal._spline, scipy.signal._sosfilt, scipy.signal._peak_finding_utils, pycocotools._mask, _cffi_backend, sklearn.__check_build._check_build, sklearn._cyutility, sklearn.utils._isfinite, sklearn.utils.sparsefuncs_fast, sklearn.utils.murmurhash, sklearn.utils._openmp_helpers, sklearn.metrics.cluster._expected_mutual_info_fast, sklearn.metrics._dist_metrics, sklearn.metrics._pairwise_distances_reduction._datasets_pair, sklearn.utils._cython_blas, sklearn.metrics._pairwise_distances_reduction._base, sklearn.metrics._pairwise_distances_reduction._middle_term_computer, sklearn.utils._heap, sklearn.utils._sorting, sklearn.metrics._pairwise_distances_reduction._argkmin, sklearn.metrics._pairwise_distances_reduction._argkmin_classmode, sklearn.utils._vector_sentinel, sklearn.metrics._pairwise_distances_reduction._radius_neighbors, sklearn.metrics._pairwise_distances_reduction._radius_neighbors_classmode, sklearn.metrics._pairwise_fast, sklearn.preprocessing._csr_polynomial_expansion, sklearn.preprocessing._target_encoder_fast, zstandard.backend_c, ormsgpack.ormsgpack, numba.core.typeconv._typeconv, numba._helperlib, numba._dynfunc, numba._dispatcher, numba.core.runtime._nrt_python, numba.np.ufunc._internal, numba.experimental.jitclass._box, google._upb._message, kiwisolver._cext, xxhash._xxhash, embreex.rtcore, embreex.rtcore_scene, embreex.mesh_construction, lxml._elementpath, lxml.etree, shapely.lib, shapely._geos, shapely._geometry_helpers, skimage.measure._ccomp, PIL._imagingcms, msgpack._cmsgpack, scipy.io.matlab._mio_utils, scipy.io.matlab._streams, scipy.io.matlab._mio5_utils, cuda_utils, __triton_launcher (total: 232)

**Comment by Tavris1** [comment]
`Update Easy-Install` has nothing to do with this error.
It is more likely caused by updating ComfyUI using `Update ComfyUI.bat`.

The solution is to roll back to a previous version of ComfyUI via **Menu > Advanced > Versions**.
Alternatively, you could try an appropriate option such as `--disable-smart-memory`.

**Comment by Eric-EWeb** [comment]
Adding the `--disable-smart-memory` flag made it work!

I am still stumped, why I have to add the flag to make it work.

But still, at least now the workflows are running without a hitch.

Thanks a lot!
